src/Computer.js

```diff
@@ -0,0 +1,81 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+import Lotto from "./Lotto.js";
+import { LOTTO_PICK, MIN_PURCHASE_AMOUNT, WIN_PRIZE } from "./constant.js";
+
+class Computer {
+  printPurchaseCount(purchaseAmount){
+    this.purchaseCount = Math.floor(purchaseAmount / MIN_PURCHASE_AMOUNT);
+  
+    Console.print(`\n${String(this.purchaseCount)}개를 구매했습니다.`);
+  }
+
+  makeLottoList(){
+    let currentPurchaseCount = 0;
+    let lottoList = [];
+    
+    while (currentPurchaseCount < this.purchaseCount) {
+      lottoList.push(Computer.buyLotto());
+      currentPurchaseCount += 1;
```

Comment:
반복문도 좋지만 이런 로직은 배열 메소드로 구현해보는 건 어떤가요? 배열 메소드를 사용하면 depth가 깊어지지 않고, 불변성을 유지하면서 배열을 조작할 수 있다는 장점이 있다고 생각해서 저는 배열 메소드를 적극 활용하거든요  🙂
```suggestion
const lottoList = new Array(this.purchaseCount).fill().map(_ => Computer.buyLotto());

```

만약 반복문을 사용한다면 이 경우에는 for문을 사용하면 더 간결해질 것 같아요!

Reply:
체이닝을 사용하여 뎁스가 더 깊어지지 않고 깔끔한 코드가 되었네요! 좋은 제안인 것 같습니다 :)